cetmix_tower_server/models/cx_tower_server.py

```
@@ -927,8 +928,37 @@ def run_command(self, command, path=None, sudo=None, ssh_connection=None, **kwar
                 error=_("Another instance of the command is already running"),
                 **log_vals,
             )
+            if not self.env.registry.in_test_mode():
+                # Commit the current transaction before starting the command.
+                # This prevents race conditions where the same command could be
+                # started again on the same server by another user before the
+                # transaction is committed. By committing here, the "is_running"
+                # flag and log record become immediately visible to other
+                # transactions, enforcing the parallel run restrictions.
+                self.env.cr.commit()  # pylint: disable=E8102
```

Pylint (3.3.7)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'E8102'

(W0012)